### PR TITLE
Adding Single Config File + Update DID-Method package to accept SideTreeNodeConfigurations as Parameter

### PR DESCRIPTION
**Pull request by ipbyrne**
This PR hooks up the `config` file inside the `dashboard` package to a single .env file.

Since NextJS is handling the loading of environment variables it is important to understand this part of the documentation: https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser

Any environment that has `NEXT_PUBLIC_` at the start of its name WILL be loaded into the client.

If the environment variable does not include `NEXT_PUBLIC_`, it will only be available inside the Node.js environment. The keys in the `config` file will still come through on the client, however, their values will always be `undefined`.

It also updates the `did-method` package so the `dashboard` package can pass in the Sidetree Node Configurations vs loading from the JSON file.

### CODE REVIEW

File: packages/dashboard/.env.example [hunk 1]
```diff
@@ -0,0 +1,10 @@
+NEXT_PUBLIC_SIDETREE_METHOD="example:sidetree.testnet"
```
Comment (OR13) [inline packages/dashboard/.env.example:1]: @ipbyrne where is the Web3 / Infura URL?

File: packages/dashboard/package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "predev": "docker-compose -f ./docker-compose.dev.yml up --detach --remove-orphans",
-    "dev": "SIDETREE_METHOD='example:sidetree.testnet' next dev",
+    "dev": "dotenv -e .env.development.local next dev",
```
Comment (OR13) [inline packages/dashboard/package.json:7]: given that we now know this works with "ganache"  locally... can we add an example of that as well?

**Review by OR13 — CHANGES_REQUESTED** [review]
Please ensure this will work with ganache / elem and not just the test node.

**Review by acarnagey — APPROVED** [review]

**Comment by ipbyrne** [comment]
> Please ensure this will work with ganache / elem and not just the test node.

Hey @OR13! I have confirmed that the env file works with both ganache/element and the side tree test node.

While running both dev options I was able to successfully create a wallet, DID and resolve the DID. I could also see the transactions/operations.

File: packages/dashboard/config/index.ts [hunk 3]
```diff
@@ -1,12 +1,41 @@
...
+  method:
+    process.env.NEXT_PUBLIC_USE_ELEMENT !== 'true'
+      ? process.env.NEXT_PUBLIC_SIDETREE_METHOD
+      : process.env.NEXT_PUBLIC_ELEMENT_METHOD,
+  description: process.env.NEXT_PUBLIC_DESCRIPTION,
   features: {
     title: 'Features',
     description: 'Decentralized Identifiers powered by Ethereum and IPFS.',
   },
+  useSideTreeNodeConfigs: process.env.NEXT_PUBLIC_USE_ELEMENT !== 'true',
```
Comment (OR13) [inline packages/dashboard/config/index.ts:14]: This confuses me

**Review by OR13 — CHANGES_REQUESTED** [review]
Please update your PR to be consistent with https://github.com/transmute-industries/sidetree.js/pull/292

**Review by OR13 — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

**Pull request merged**